modules/perception/obstacle/camera/lane_post_process/cc_lane_post_processor/cc_lane_post_processor.cc

```diff
@@ -793,6 +810,8 @@ bool CCLanePostProcessor::Process(const cv::Mat &lane_map,
           }
         }
       }
+      // left most point - C0 < 2.5
+      // *** TO DO *** use C0 or closest_x point for the distance measure.
```

Comment:
please use TODO(name) format for tasks to be done in the future.

Reply:
Thank you for the review and advice. I will follow the format!